ext/oci8/tests/bug70700.phpt

```diff
@@ -0,0 +1,182 @@
+--TEST--
+Tests for LOBs with multibyte strings, reading them out in chunks
+(Doc Bug #70700)
+--CREDITS--
+Chuck Burgess
+ashnazg@php.net
+--SKIPIF--
+<?php
```

Comment:
Please add the following check here:
if (!extension_loaded('mbstring')) die('skip mbstring is not enabled');

After adding this, we're good to go.